client/actions/index.js

```diff
@@ -64,6 +64,12 @@ export const dropAnswer = (index: number) => (dispatch: any, getState: Function)
   dispatch({ type: at.DROP_ANSWER, payload: {uid, index}})
 }
 
+
+export const dropClue = (index: number) => (dispatch: any, getState: Function) => {
```

Comment:
Since this looks pretty much identical to dropAnswer we can probably Pull it out into a generic function:

```javascript
const dropper = (actionType: string) => (index: number) => (dispatch: any, getState: Function) => {
   const uid = getDragData(getState())
   dispatch({ type: actionType, payload: {uid, index}})
 }

export const dropClue = dropper(at.DROP_CLUE)
export const dropAnswer = dropper(at.DROP_ANSWER)
```